README.md

```diff
@@ -1,2 +1,84 @@
 # world-holidays
 A Global Holiday web app that provides holiday data for 251 countries around the world.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Technologies Used
+
+- [x] React
+- [x] Redux
+- [x] Stylelint and ESlint
+- [x] VS Code
+
+## Website Mockup 📱 💻 🖥️
+![image](https://user-images.githubusercontent.com/93576145/176367874-2f605876-ee9a-4c76-8aa4-615eeb647df4.png)
+
+
+## Live Demo 🔗
+
+[World holidays](https://worldholidays.netlify.app/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+- [x] A web browser like Google Chrome
+- [x] A code editor like Visual Studio Code with Git and Node.js.
+
+### Setup
+
+Clone the repository using the GitHub [link](https://github.com/iBel251/world-holidays.git)
+
+### Install
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone https://github.com/iBel251/world-holidays.git
+```
+
+### Usage
+
+Kindly modify the files as needed.
+
+### Run tests
+
+To check for linters, use the files provided by [Microverse](https://github.com/microverseinc/linters-config). A GitHub action is also set to run during pull request.
+```
+$ npm install
+```
+
+### Deployment
+
+This app is deployed in the GitHub Pages for easy viewing upon merged on the main branch.
+Please find the link in the Live Demo section.
+
+
+## Authors
+
+👤 **Imran Belete** <img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="20"/>
+
+  - [GitHub](https://github.com/iBel251)
+  - [Twitter](https://twitter.com/ImranBel251)
+  - [LinkedIn](https://www.linkedin.com/in/imran-belete/)
+  - [Gmail](mailto:ibbimran7@gmail.com)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/iBel251/world-holidays/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+- Code Reviewers
```

Comment:
- [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).